### PR TITLE
[VIT-3007] Add At-Home Phlebotomy endpoints

### PR DESCRIPTION
**Pull request by gabrieleangeletti**

### CODE REVIEW

File: lib/utils.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export const keysToCamel = (o: any): any => {
```
Comment (andersio) [inline lib/utils.ts]: Seems like Axios should have a way to do snake to camel?

Comment (gabrieleangeletti) [reply lib/utils.ts]: No idea, can look into it :+1:

Comment (ib250) [reply lib/utils.ts]: It does, i recall using a library for this but cant remember the name now

Comment (gabrieleangeletti) [reply lib/utils.ts]: I didn't find it within Axios but I found another one called `humps` that does this. Just replaced it.

File: client/AtHomePhlebotomy.ts [hunk 2]
```diff
@@ -0,0 +1,100 @@
+import { AxiosInstance } from 'axios';
+import { camelize, camelizeKeys } from 'humps';
```
Comment (ib250) [inline client/AtHomePhlebotomy.ts:2]: Ah yeah I did use humps for this a while ago, with custom axios middleware

**Review by ib250 — APPROVED** [review]

**Pull request merged**